src/corehost/cli/deps_resolver.cpp

```diff
@@ -303,25 +313,47 @@ bool deps_resolver_t::probe_deps_entry(const deps_entry_t& entry, const pal::str
     return false;
 }
 
-bool report_missing_assembly_in_manifest(const deps_entry_t& entry)
+bool report_missing_assembly_in_manifest(const deps_entry_t& entry, bool continueResolving = false)
 {
-    trace::error(_X(
-        "Error:\n"
-        "  An assembly specified in the application dependencies manifest (%s) was not found:\n"
-        "    package: '%s', version: '%s'\n"
-        "    path: '%s'"),
-        entry.deps_file.c_str(), entry.library_name.c_str(), entry.library_version.c_str(), entry.relative_path.c_str());
+    bool showManifestListMessage = !entry.runtime_store_manifest_list.empty();
 
-    if (!entry.runtime_store_manifest_list.empty())
+    if (entry.asset_type == deps_entry_t::asset_types::resources)
     {
-        trace::error(_X(
-            "  This assembly was expected to be in the local runtime store as the application was published using the following target manifest files:\n"
-            "    %s"),
-            entry.runtime_store_manifest_list.c_str());
+        // Treat missing resource assemblies as informational.
+        continueResolving = true;
+
+        trace::info(MissingAssemblyMessage.c_str(), _X("Info"),
+            entry.deps_file.c_str(), entry.library_name.c_str(), entry.library_version.c_str(), entry.relative_path.c_str());
+
+        if (showManifestListMessage)
+        {
+            trace::info(ManifestListMessage.c_str(), entry.runtime_store_manifest_list.c_str());
+        }
     }
+    else if (continueResolving)
+    {
+        trace::warning(MissingAssemblyMessage.c_str(), _X("Warning"),
```

Comment:
No since the previous code logged a warning before here https://github.com/dotnet/core-setup/blob/master/src/corehost/cli/deps_resolver.cpp#L586. Now it just calls this same helper method to ensure to ensure it logs an info for missing satellite assemblies, and a warning for other assemblies. As a benefit, however, it also outputs with the improved error message format.